monitor/host_flagging.go

```diff
@@ -128,6 +128,20 @@ func flagIdleHosts(d []distro.Distro, s *evergreen.Settings) ([]host.Host, error
 	return idleHosts, nil
 }
 
+// getCommuncationTime will find the last time this host has communicated with evergreen or vice versa
+func getCommuncationTime(h host.Host) time.Duration {
+	if h.LastCommunicationTime.After(h.CreationTime) {
+		return time.Since(h.LastCommunicationTime)
+	}
+	if h.StartTime.After(h.CreationTime) {
+		return time.Since(h.StartTime)
+	}
+	if !h.LastCommunicationTime.IsZero() {
+		return time.Since(h.LastCommunicationTime)
+	}
+	return time.Since(h.CreationTime)
+}
```

Comment:
write a test for this? we should deploy *now* and maybe have the test lag? 

why not move this into the host package as a method on host?

Reply:
Done. I'll write a test later